homeworks/hw12/src/bank.rs

```diff
@@ -0,0 +1,869 @@
+use log::{debug, error, info};
+/// This module contains the implementation of a simple banking system.
+///
+/// The `Bank` struct represents a bank and provides methods for managing accounts
```

Comment:
```suggestion
/// The [`Bank`] struct represents a bank and provides methods for managing accounts
```

Можно сделать ссылочку сразу на структуру таким образом)